MayssaraOmar/assignment3/Word search/src/Word search.h

```diff
@@ -0,0 +1,11 @@
+
+
+#ifndef WORD_SEARCH_H_
+#define WORD_SEARCH_H_
+
+#include <unordered_set>
+#include "dictionary.h"
+
+void get_words_from_grid(vector<vector<char> > &grid, dictionary &dic,unordered_set<string> &ans);
```

Comment:
A comment here exampling what the function does and what it expects as arguments would be useful for someone wanting to use this code.